qt/widgets/mplcpp/src/MantidColorMap.cpp

```diff
@@ -28,8 +28,8 @@ namespace MplCpp {
  */
 QString MantidColorMap::chooseColorMap(const QString &previous,
                                        QWidget *parent) {
-  static QStringList allowedCMaps{"coolwarm", "jet", "summer", "winter",
-                                  "viridis"};
+  static QStringList allowedCMaps{"coolwarm", "gray",   "jet",    "plasma",
```

Comment:
Can you remove the extra spaces?

Reply:
The extra spaces were added by `clang-format` to make the elements in the list line up on the top and bottom lines of the list